src/Component/Footer/TodosFilter.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import '../../styles/todo-list.css';
+import '../../styles/index.css';
+import '../../styles/filters.css';
+
+const handleFilter = (event, callFilter, criteria) => {
+  event.preventDefault();
+  callFilter(criteria);
+};
+
+export const TodoFilter = (props) => {
+  const [todosCompletedCount, settodosCompletedCount] = useState(0);
+  const [isAnyCompleted, setIsAnyCompleted] = useState(0);
+
+  const {
+    todos,
+    filter,
+    onFilter,
+    onRemoveCompleted,
+    showFotter,
+  } = props;
+
+  const filterCompleted = todos
+    .filter(todo => todo.completed === false)
+    .length;
+
+  useEffect(() => {
+    settodosCompletedCount(filterCompleted);
+  }, [filterCompleted]);
+
+  const filterIsAnyCompleted = todos
+    .some(todo => todo.completed === true);
+
+  useEffect(() => {
+    setIsAnyCompleted(filterIsAnyCompleted);
+  }, [filterIsAnyCompleted]);
+
+  return (
+
+    <footer
+      className={
+        classNames({
+          footer: showFotter,
+          hidden: !showFotter,
+        })}
+    >
+      <span className="todo-count">
+        {`${todosCompletedCount}  items left`}
+      </span>
+
+      <ul className="filters">
+        <li>
+          <a
+            href="#/"
+            className={classNames({ selected: (filter === 'all') })}
+            onClick={event => handleFilter(event, onFilter, 'all')}
```

Comment:
create constant with filter's values
so you can import it and use it like this `FILTERS.all`